src/ddbToS3/dynamodb.ts

```diff
@@ -0,0 +1,71 @@
+import AWS from 'aws-sdk';
+import _ from 'lodash';
+import type { BatchStatementRequest } from 'aws-sdk/clients/dynamodb';
+
+const DEFAULT_BATCH_SIZE = 10;
+const BATCH_SIZE = parseInt(process.env.DYNAMODB_BATCH_SIZE || DEFAULT_BATCH_SIZE.toString(), 10);
+const TABLE_NAME = process.env.RESOURCE_TABLE || 'table';
+
+export default class DynamoDB {
+    private getStatements = (records: any[], ttlsInSeconds: Map<string, number>): BatchStatementRequest[] => {
+        const statements = new Set<string>();
+        const now = Math.floor(Date.now() / 1000);
+
+        records.forEach((record) => {
+            const resourceType = record.dynamodb.NewImage.resourceType.S;
+            const ttl = now + ttlsInSeconds.get(resourceType)!;
```

Comment:
Sorry, what gets set here if there is no configured TTL for the provided resourceType?

Reply:
'undefined' would be set. But there always be TTL for the resourceType. So It will never be undefined.